static/assert.ts

```diff
@@ -67,7 +67,7 @@ function fail(fail_message: string, user_message: string | undefined, args: any[
     }
 }
 
-export function assert<C>(c: C, message?: string, ...extra_info: any[]): asserts c {
```

Comment:
This is a TS bug, discussed here: https://github.com/microsoft/TypeScript/issues/60130